client/src/components/DataFiles/DataFilesModals/DataFilesDownloadMessageModal.module.scss

```diff
@@ -0,0 +1,3 @@
+p {
+  font-weight: bold;
+}
```

Comment:
I think in order for css modules to work, we need to use class names. Could you make this a class instead, or nest this inside a `root` class?